jest-setup.js

```diff
@@ -89,72 +89,91 @@ global.testRule = (rule, schema) => {
                 };
 
                 return stylelint(options).then(output => {
-                  const warning = output.results[0].warnings[0];
+                  const actualWarnings = output.results[0].warnings;
 
-                  expect(output.results[0].parseErrors).toEqual([]);
-                  expect(testCase).toHaveMessage();
-
-                  if (testCase.message !== undefined) {
-                    expect(_.get(warning, "text")).toBe(testCase.message);
+                  if (
+                    schema.fix &&
+                    testCase.warnings &&
+                    testCase.warnings.length > 1
+                  ) {
+                    throw new Error(
+                      "Several expected warnings can not be used with { fix: true } in schema yet"
```

Comment:
Don't know how to implement it better

Reply:
This is not needed. With `--fix` all warnings are fixed.

Reply:
So you suggest to allow several `fixed` in `warnings` ?

Reply:
Based on how `fix` works we'll need only one `fixed` per case.

Reply:
done